### DISCUSSION TITLE
Copilot doesn't play well with regular autocomplete/Intellisense

### DISCUSSION THREAD

**Discussion by zenhorace** [community/community#11156] [Copilot Conversations]
I'm running into an issue while using Copilot, where it seems to compete with regular autocomplete/IntelliSense extension for the language. For example, when using Go, if I do `fmt.` I either get the list of methods on that package as usual _or_ I get some suggestion by Copilot. In the case of the latter, if the function that Copilot selected isn't what I want (or if I actually want to scan available methods), I have to do a quick sequence of <kbd>esc</kbd> ->  <kbd>backspace</kbd> (to delete the period) -> <kbd>.</kbd> and hope I do it fast enough that IntelliSense shows up before Copilot reengages.

If there's a better way, please let me know 🥺 
If there isn't, please consider having Copilot play well with existing IntelliSense 🙏🏽

**Comment by jansorg** [comment]
@zenhorace Thanks for your feedback! Which editor or IDE are you using?

**Reply by zenhorace** [reply]
Whoops. Forgot to mention that. VScode.

**Comment by RedBreadcat** [comment]
I have the same quibble too, using VSCode with the rust-analyzer extension. I've noticed that it's a bit of a race condition. If Copilot makes its suggestion first, then I won't get regular autocomplete suggestions. However if I'm getting autocomplete suggestions, Copilot won't make any recommendations.

**Comment by sonic1981** [comment]
Plenty of people already mentioned this. Prob upvote one of those. https://github.com/github/feedback/discussions/9430

**Reply by MarkBrainfire** [reply]
Perhaps, but evidently not enough listening and acting on it.

**Comment by MarkBrainfire** [comment]
Visual Studio 2022 ... ditto.  I want to see the members of the class, but Copilot prevents me.

**Comment by RawToast** [comment]
Same issue on Visual Studio Code, I use the same solution... I've seen the same issues on IntelliJ

**Comment by larshaugeli** [comment]
I have the same problem with VS 2022. Is there any plan for fixing this? Copilot is pretty useless if we can't access our built in IntelliSense...

**Comment by michdim** [comment]
@zenhorace, thank you for sharing your workaround! It has been quite helpful. I've discovered a slight variation that seems to work consistently without the need for rapid typing: first delete the`.`, then press `esc` to hide copilot suggestion , and then type `.` again. This always brings up the IntelliSense/autocomplete suggestions list for me. I appreciate your contribution, as it has significantly improved my experience using Copilot alongside built-in autocomplete features in VScode.

**Comment by foxt** [comment]
Annoyingly, Copilot X also steals the Cmd+I keybinding, which *used* to open the intellisense menu.

**Comment by cinco-de-mayonnaise** [comment]
It's amazing that it has been more than 1 year since and this still has not been addressed. My only option is to not use Copilot on a per-workspace basis.

**Comment by kwood-gmr** [comment]
This is still a MAJOR issue. Basically - copilot is giving suggestions that don't match names of stuff in my code and IntelliSense is giving NOTHING at all.

**Comment by john-walteriii** [comment]
Backspace on the dot followed by escape involves both pinkies followed by an awkward re-dot. Much better would be single tab for intellisense and double tab for copilot.
